### ISSUE TITLE
Local variable declarations and assignments should be preceded by a blank line

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#364]
A declaration (and assignment) to a local variable should be precededby a blank line if the preceding line does not contain another variable declaration/assignment. The reason is ease of reading  (spotting variable declarations with ease).

Following _should report_ a violation:
```c#
DoSomething();
var x = 42;
var y = "something";
DoSomethingElse(x, y);
```

While following _should **NOT** report_ a violation:
```c#
DoSomething();

var x = 42;
var y = "something";
DoSomethingElse(x, y);
```

In addition, following _should report_ a violation:
```c#
var x = 42;

DoSomething();
x = 0815;
DoSomethingElse(x, y);
```

While following _should **NOT** report_ a violation:
```c#
var x = 42;

DoSomething();

x = 0815;
DoSomethingElse(x, y);
```
